helper.go

```diff
@@ -48,6 +49,56 @@ func ImageExists(filename string) bool {
 	return !info.IsDir()
 }
 
+// Check for remote filepath, e.g: https://test.webp.sh/node.png
+// return StatusCode, etagValue
+func GetRemoteImageInfo(fileUrl string) (int, string) {
+	res, err := http.Head(fileUrl)
+	if err != nil {
+		log.Fatal("Connection to remote error!")
+	}
+	if res.StatusCode != 404 {
+		for index, _ := range res.Header {
```

Comment:
Why loop into headers? Shouldn't we just use `res.Header.Get("etag")` instead?

Reply:
* Aliyun OSS, Tencent COS and Nginx has header `ETag`
* AWS has header `etag`

We need to get the wanted value, regardless of the remote backend.

Reply:
It doesn't matter. `res.Header.Get("etag")` and `res.Header.Get("ETag")`, and even `res.Header.Get("EtAg")` will get the same result for you.